assets/src/scss/blocks/PostsListBlock.scss

```diff
@@ -0,0 +1,143 @@
+.posts-list-query {
+  .wp-block-post-template.is-layout-flow li {
+    width: 100% !important;
+  }
+
+  .wp-block-columns.is-layout-flex {
+    gap: $sp-2x;
+  }
+
+  .wp-block-post-featured-image {
+    flex-basis: 30%;
+
+    img {
+      @include medium-and-up {
+        height: 180px;
+      }
+
+      @include large-and-up {
+        height: 210px;
+      }
+      width: 100% !important;
+      object-fit: cover;
+    }
+  }
+
+  .wp-block-group.is-layout-flow {
+    flex-basis: 70%;
+  }
+
+  .wp-block-post-title {
+    font-weight: $font-weight-bold;
```

Comment:
You can use tokens instead!

Reply:
That is a token from [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_tokens.scss#L114), I can use the css variable instead if that is what you mean.